config/src/lib.rs

```diff
@@ -169,3 +198,34 @@ pub trait Configurable: Serialize + DeserializeOwned {
     /// Returns documentation for all fields in form of json object
     fn get_docs() -> Value;
 }
+
+/// Json config for getting configuration
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum GetConfiguration {
```

Comment:
I'm not sure they belong here. Maybe in `iroha_data_model`?

Reply:
ah, I placed them here for a good reason. This was giving me headaches last night.
If `PostConfiguration` is placed in `data_model`, then `logger::Level` has to follow it there which means that `logger` crate will get a dependency to `iroha_data_model`. But that's silly because `logger` has nothing to do with `data_model`. However, all crates(including `client`) share the `config` crate so this makes a natural fit for config get/set request related structures.

Consequently, `GetConfiguration` goes where `PostConfiguration` goes

Reply:
`iroha_logger` can be added as `data_model` dependency if needed, but not transitively